openmetadata-ui/src/main/resources/ui/webpack.config.dev.js

```diff
@@ -116,6 +116,7 @@ module.exports = {
     },
     alias: {
       process: 'process/browser',
+      Quill: path.resolve(__dirname, 'node_modules/quill'),
```

Comment:
Can we please add comments in the code on why we need this?

Reply:
Done, added comment for this.